packages/batch-submitter/hardhat.config.ts

```diff
@@ -6,6 +6,8 @@ import {
   RUN_OVM_TEST_GAS,
 } from './test/helpers/constants'
 
+import '@nomiclabs/hardhat-ethers'
```

Comment:
Is there anything wrong with adding this import? Does it make the production version of our software use `hardhat`?

Reply:
I believe it just mutates the `hre`